VSFSMenuCommand/VSFSMenuCommand.fsproj

```diff
@@ -140,21 +137,17 @@
     <Reference Include="System.Configuration" />
     <Reference Include="System.Core" />
     <Reference Include="System.Design" />
-    <Reference Include="System.Windows.Forms" />    
-    
+    <Reference Include="System.Windows.Forms" />
     <Reference Include="Microsoft.VisualStudio.OLE.Interop" />
     <Reference Include="Microsoft.VisualStudio.Shell.Interop" />
     <Reference Include="Microsoft.VisualStudio.Shell.Interop.8.0" />
     <Reference Include="Microsoft.VisualStudio.Shell.Interop.9.0" />
     <Reference Include="Microsoft.VisualStudio.Shell.12.0" />
     <Reference Include="Microsoft.VisualStudio.Shell.Immutable.10.0" />
-    <Reference Include="EnvDTE, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
-    <Reference Include="EnvDTE80, Version=8.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
```

Comment:
IMO the whole point of this exercise is to give newbies the minimum they need to follow/understand. So _any_ extraneous fluff is bad. So what if they possibly might need other things in the future?

In the next more advanced sample you'd add those in, if and as you need them. This also helps newbies to see what has changed.

Same goes for any MEF stuff you added in here since MEF is _not_ involved in a Menu Command.

Instead do a MEF only sample, and omit any VSPackage stuff that it wouldn't need.